backend/data/src/main/kotlin/io/tolgee/development/testDataBuilder/data/NotificationsTestData.kt

```diff
@@ -3,13 +3,19 @@ package io.tolgee.development.testDataBuilder.data
 import io.tolgee.model.notifications.Notification
 
 class NotificationsTestData : BaseTestData() {
+  private var currentTaskNumber = 1L
+
+  init {
+    user.username += "@example.com"
```

Comment:
Afaik BaseTestData do have the username as constructor param.

Reply:
Ah, they do! Thanks, I'll change it.